### ISSUE TITLE
Editor Guideline

### ISSUE THREAD

**Issue by blue-devil** [mrfearless/RadASM2#17]
RadASM2 does not have an `Editor GuideLine` feature to set a line on a specific column on code editor. 

How To:
- This can be handled in `Option > Code Editor Options`
    - a checkbox to set editor guideline.
    - One up-down control for column number
    - one color picker control for setting color
- The settings then be saved in RadASM.ini

**OR**

While writing this issue i have realized that maybe this can be handled via a plugin?

**Comment by mrfearless** [comment]
Yes i guess either option would work, the plugin option sounds good, but not sure if RadASM exposes all the information required to implement it, so might have to do it in the main code.

**Comment by blue-devil** [comment]
I have opened a thread on MAS32 forums.

https://masm32.com/board/index.php?topic=10381.0

Maybe we can get some info from KetilO, so I can speed up research about this.
